packages/modules/vehicles/bmwbc/api.py

```diff
@@ -22,7 +22,7 @@
 from modules.common.store import RAMDISK_PATH
 from pathlib import Path
 
-dataPath = "/var/www/html/openWB/data/bmwbc"
+DATA_PATH = "data/bmwbc"
```

Comment:
Es ist einfacher, wenn Du mit dem Path-Objekt arbeitest.
```suggestion
DATA_PATH = Path(__file__).resolve().parents[4] / "data" / "bmwbc"
```

Reply:
Ich habe es bei mir so geändert, getestet und ein neues commit gemacht.